tutorials/metrics/metrics_file_regex_with_invalid_linenum.yml

```diff
@@ -0,0 +1,15 @@
+buildspecs:
+  metric_file_regex:
+    executor: generic.local.bash
+    type: script
+    description: capture result metric from file path
+    run: echo -e "HPCG result is INVALID with a GFLOP/s rating of=28.1215\n
```

Comment:
see if this works, i know it works in shell.

You should use the `|` symbol see https://yaml.org/spec/1.2.2/#literal-style which allows one to literal block symbol

```
run: |
  cat <<EOF > hpcg.txt
  HPCG result is INVALID with a GFLOP/s rating of=28.1215 
  HPCG result is VALID with a GFLOP/s rating of=68.9888
  EOF

Reply:
here you can do it this way

```
echo -e "HPCG result is INVALID with a GFLOP/s rating of=28.1215" > hpcg.txt
echo -e "HPCG result is VALID with a GFLOP/s rating of=68.9888 >> hpcg.txt
```

Reply:
```
run: |
  echo -e "HPCG result is INVALID with a GFLOP/s rating of=28.1215" > hpcg.txt 
  echo -e "HPCG result is VALID with a GFLOP/s rating of=68.9888" >> hpcg.txt
```
works